### PR TITLE
javascript/javascript1/week3

### PR DESCRIPTION
**Pull request by ehsanesmaeil**

### CODE REVIEW

File: javascript/javascript1/week3/w3-exercise-2.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+
+
+
+let speedKmPerH = 50;
+let distanceKm = 155;
+
+function hourToGo() {
+  let hour = Math.trunc(distanceKm / speedKmPerH);
```
Comment (sowmya1408) [inline javascript/javascript1/week3/w3-exercise-2.js:9]: Variable naming conventions , hour/minute - is a general naming, be very specific.
ex: travelInHour etc...

File: javascript/javascript1/week3/w3-exercise-3.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  },
+  {
+    title: "Modern Family",
+    days: 11,
+    hours: 14,
+    minutes: 46,
+  },
+];
+
+let age = 80;
```
Comment (sowmya1408) [inline javascript/javascript1/week3/w3-exercise-3.js:28]: you can use const keyword here, as you are not reassign this variable

**Review by sowmya1408 — CHANGES_REQUESTED** [review]
Good job!!
Use specific variable naming convention , not in general as min, hour etc...

File: javascript/javascript1/week3/w3-exercise-1.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+console.log(names);
+// ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+
+
+// second way :
+ 
+const names = ['Peter', 'Ahmad', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala'];
+  
+ function deleteName(name){
```
[the remark this answers is not in the document]
Comment (ehsanesmaeil) [reply javascript/javascript1/week3/w3-exercise-1.js:28]: Thanks for the review.

**Pull request merged**